<launch>
    <!-- My Package launch file -->
    <node pkg="topic_publisher_pkg" type="topic_publisher" name="topic_publisher"  output="screen">
        <param name = "linear_x" type = "double" value = "0.4" />
        <param name = "angular_z" type = "double" value = "0.4" />
    </node>
</launch>